src/ethereum/EthHashInfo/index.tsx

```diff
@@ -54,6 +54,7 @@ type Props = {
   textSize?: ThemeTextSize;
   showAvatar?: boolean;
   customAvatar?: string;
+  customAvatarFallback?: string;
```

Comment:
Is this image always going to be the `<>` icon? If so, we could just store the SVG in the repo. Or at least use it as the default value.

Reply:
There are 2 different repositories. 
`<>` is needed at this moment in SRC, but as this lib is also helpful for SApp I would not use it as a default one.